docs/openshift.md

```diff
@@ -0,0 +1 @@
+WIP
```

Comment:
should this document have some content before we merge this PR? Or else delete it?

Reply:
agree 
but since we are quite active on productization, I would expect content soon.
If it not happened or if it will bring other documetntation structure - will delete it.